Add issue template for Troubleshooting and FAQ (#1555)
fermyon/spin: .github/ISSUE_TEMPLATE/faq-and-troubleshooting-sources.md

---
name: FAQ and Troubleshooting sources

> about: This template is supposed to provide FAQ and Troubleshooting materials.

Reviewer: ```suggestion
about: This template provides links to the FAQ and Troubleshooting materials.
```
